tests/dtm/test_single_process_restart.py

```diff
@@ -740,3 +739,110 @@ def test_copy_object_during_m0d_restart(self):
             assert_utils.assert_true(resp, "Checksum validation Failed.")
         self.test_completed = True
         self.log.info("ENDED: Verify copy object during m0d restart using pkill.")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42253")
+    def test_copy_object_after_rgw_restart(self):
+        """Verify copy object after rgw restart using pkill."""
+        self.log.info("STARTED: Verify copy object after rgw restart using pkill")
+        object_name = 'object-test-42253'
+        self.log.info("Step 1: Start write Operations :")
+        for i in range(0, 2):
+            resp = self.s3_obj.create_bucket(bucket_name=f"bucket-test-42253-{i}")
+            assert_utils.assert_true(resp[0], resp[1])
+        bucket_list = self.s3_obj.bucket_list()[1]
```

Comment:
This might list the preexisting buckets also for user. 
You can maintain a list of bucket name post creation .

Reply:
changed to local list